src/routes/docs/quick-starts/+page.svelte

```diff
@@ -127,7 +127,7 @@
 	<meta name="twitter:card" content="summary_large_image" />
 </svelte:head>
 
-<main class="aw-main-section">
+<main class="aw-main-section" id="main">
```

Comment:
Adding it to pages alone won't be enough, we should add them to all relevant layouts